proto/messages.proto

```diff
@@ -28,15 +28,19 @@ message GenericMessage {
 }
 
 message CompositeMessage {
-  repeated content {
+  repeated Item items = 1;
+}
+
+message Item {
```

Comment:
This type is very specific for composite message, can we call it `CompositeMessageContent` instead of `Item`?

Reply:
or `CompositeMessageContentItemThatCanBeAnythingButAssetOrCallingStuff`